science/openmpi/Portfile

```diff
@@ -136,15 +137,13 @@ if {${subport} != ${name}} {
     archive_sites
 
     depends_build-append        port:pkgconfig
+    depends_build-append        port:perl5
```

Comment:
I suspect this is no longer needed, due to changes made to perl5 since you first submitted this. Can you remove this line ?

Reply:
Yup, I can confirm that the port builds without this line.